docs/pywbemcli/commands.rst

```diff
@@ -1559,6 +1563,77 @@ Example:
 See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
 command.
 
+.. _`Server schema command`:
+
+``server schema`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``server schema`` command returns information on the  :ref:`Schema`
+defined for classes in the namespace(s) of the :term:`current connection`
+WBEM server.
+
+The overview reports a summary by namespace of this information and the
+detail view (``server schema --detail`) reports the information for each
+:ref:`Schema` in the namespace.
+
+For each CIM schema in each namespace, it provides information on the
+:ref:`Schema` release version  whether the :ref:`Schema` has experimental
+classes, and the number of classes in the :ref:`Schema`.
+
+Pywbemcli determines the version of the :ref:`Schema` by finding the highest
+value of the Version qualifier on any of the classes in the namespace that
+are in that :ref:`Schema`
+
+For the :ref:`DMTF schema` in the overview report the highest value is
+determined by finding the highest version value for any of the DMTF schemas
+(normally 'CIM', or 'PRS'). The highest value anywhere in the namespace should
+be the :ref:`DMTF schema` package that was used to define the qualifier
+declarations and the CIM/PRS classes in that namespace.
+
+Example:
+
+.. code-block:: text
+
+    $ pywbemcli --name myserver server schema
+
+    Schema information; namespaces: all;
+    Namespace                      Schemas              classes  DMTF schema    experimental
```

Comment:
Given the terms in DSP0004, the column heading "DMTF schema" should be changed to "CIM schema".

Reply:
DONE